docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,228 @@
+
+Other Labels can also be added by reviewers at their discretion. 
+
+## 5. Rebasing
+Rebasing is used when there are conflicts between your branch and the branch your PR is based off of (an alternative to merging). 
+Here is a general workflow:
+
+    cd /var/www/loris
+    git checkout branch_name
+    git fetch aces
+    git rebase aces/main
+    (#) while (conflicts)  { 
+        // fix any conflicts and add any files
+        git add  filename
+        git rebase --continue
+        // OR you can "skip" specific changes
+        git rebase --skip
+    # } loop back to (#) - fix any remaining conflicts.... until no more conflicts
+    git push --force-with-lease my_fork branch_name
+
+### Fixing a rebase gone wrong
+1. The most basic way of “fixing” a rebase gone wrong is to go back to the commit right before the rebase. The easiest way to do this is to either reset or revert to a previous commit. **Note** that this is a potentially destructive command, so be very cautious when using it. It is also recommended that you make a backup of your branch before performing these commands so that you don't lose any changes. 
+For example:   
+    ```
+    git reset --hard <commit_hash>
+    git push --force-with-lease my_fork branch_name
+    ```   
+
+2. A more complicated tool that can be used to solve bigger rebasing errors is the command [`git cherry-pick`](https://git-scm.com/docs/git-cherry-pick). This command allows you to pick specific commits to apply to a branch. For example, if you rebase the wrong branch or your PR is incorrectly displaying all the commits or files from the branch you rebased, this tool will come in handy. Here is an example workflow:   
+    ```
+    git checkout branch_name          # The branch with rebasing errors
+    git branch -m "temp_branch_name"  # Change the branch name locally:
+                                      # This is a failsafe since you will
+                                      # be changing the history of the
+                                      # branch remotely
+    git fetch aces
+    git checkout aces/main            # Checkout the branch that your PR is
+                                      # based off of
+    git checkout -b "branch_name"     # Create a “new” branch with the same
+                                      # name as the original branch
+    git cherry-pick <commit-hash>     # Cherry-pick the commits that you would like to keep
+    git push --force-with-lease my_fork "branch_name"
+    ```
+
+## 6. Accessing a Database
+To access the database, use the following command with the credentials given to you and provide the password on prompt:  
+```
+mysql -u user -h host database -p
+```
+
+If you are new to the MySQL commandline, try using `\G` instead of `;` at the end of a query - this will print one column value per line, which makes the query output easier to read.
```

Comment:
I really don't think it's our job to explain this in any way. and if iit was, this would not be the place for it. by the time you get to developing you must have installed LORIS already and there are 100 places where you would have already connected.

In general we can't write documentation in silos, this is part of a larger documentation and should not reiterate this kind of details